DFC.ServiceTaxonomy.GraphVisualiser/Controllers/VisualiseController.cs

```diff
@@ -116,6 +129,28 @@ public async Task<ActionResult> Data([FromQuery] string? contentItemId, [FromQue
             return await GetData(contentItemId, graph);
         }
 
+        public async Task<ActionResult> NodeLink ([FromQuery] string nodeId, [FromQuery] string route, [FromQuery] string graph)
+        {
+            if (graph!.ToLowerInvariant() == "published")
```

Comment:
You could use `IContentItemVersionFactory` to get the right `IContentItemVersion`.